### PR TITLE
typescript migration

### PR DESCRIPTION
**Pull request by makarofalex3**
started typescript migration

### CODE REVIEW

**Review by vkomodey — CHANGES_REQUESTED** [review]
Do we need to update run instructions here(both local and remote in the docker)?

Also, I see a bunch of files with js and jsx extensions. Do we really need to preserve these extensions?

File: src/theme.tsx [hunk 1]
```diff
@@ -6,8 +6,9 @@ const theme = createTheme({
       main: '#6309ff',
       light: '#616161',
       dark: '#4a148c',
-      mediumLight: '#ba68c8',
-      white: '#f5f5f5',
+      // TO-DO delete theme dont have this values
+      // mediumLight: '#ba68c8',
+      // white: '#f5f5f5'
```
[the remark this answers is not in the document]
Comment (makarofalex3) [reply src/theme.tsx:11]: yes we can!

File: src/utils/index.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * @param {*} escrowFactories Array of escrow Factories
  * @returns the total count of factory
  */
-export const countEscrowFactory = (escrowFactories) => {
+export const countEscrowFactory = (escrowFactories: Array<any>) => {
```
[the remark this answers is not in the document]
Comment (makarofalex3) [reply src/utils/index.ts]: added

File: src/theme.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (makarofalex3) [reply src/theme.tsx]: fixed

**Comment by vkomodey** [comment]
What about running instructions(both local and remote)? Are they the same as were before?

**Pull request merged**